src/features/SelectGroup/SelectGroup.scss

```diff
@@ -0,0 +1,13 @@
+.react-select__control {
+  background-color: var(--primary-background-color) !important;
```

Comment:
There is no reason to use `!important` here, because you can overwrite component's styles by simple increasing classes specificity